### ISSUE TITLE
Codecov badge shows 0% coverage

### ISSUE THREAD

**Issue by dmitrytokarev** [kubernetes-client/python#1532]
**Link to the issue (please include a link to the specific documentation or example)**:
https://github.com/kubernetes-client/python#readme

**Description of the issue (please include outputs or screenshots if possible)**:
codecov badge shows 0% code coverage.
To fix navigate to https://app.codecov.io/gh/kubernetes-client/python/settings
copy markdown example and paste it to the README.md
official instructions: https://about.codecov.io/product/feature/badges/

**Comment by dmitrytokarev** [comment]
I would fix myself, but don't have access to see that codecov settings page.

**Comment by roycaihw** [comment]
Thanks for reporting the issue!

According to the badge link in the codecov settings page, the reported coverage seems to be actually 0%. On https://app.codecov.io/gh/kubernetes-client/python/pulls, I see error messages "Missing base report" and "There was an error processing coverage reports". I wonder if we broke some of the [codecov configs](https://github.com/kubernetes-client/python/pull/54) when we moved from Travis CI to Github Actions.
